jsettlers.graphics/src/main/java/jsettlers/graphics/map/MapContent.java

```diff
@@ -374,6 +379,17 @@ private float messageAlpha(IMessage m) {
 				: Math.max(0, 1f - (float) age / IMessage.MESSAGE_TTL);
 	}
 
+	private void drawWinStateMsg(GLDrawContext gl) {
+		if(localPlayer.getWinState() == EWinState.UNDECIDED) {
+			return;
+		}
+		Color color = (localPlayer.getWinState() == EWinState.WON ? Color.GREEN : Color.RED);
```

Comment:
Why do you have the parenthesis around the ternary? They should not be needed.

Reply:
I don't know if I have an answer for that - that's the way I tend to write them. Feel free to remove it. ;) Or should I commit the fix?